pkg/dogstatsd/replay/capture.go

```diff
@@ -14,6 +14,11 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/config"
 )
 
+const (
+	// GUID will be used as the GUID during capture replays
+	GUID = 999888777
```

Comment:
Can we expand the comment to explain why this number was chosen ?

Reply:
it's just a magic number, but I can provide some context